Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+  return (
+    <img src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png"/>
```

Comment:
You can also include an *alt* attribute for better accessibility and to default to in case the image fails to load.
https://www.w3schools.com/tags/att_img_alt.asp